datasource.go

```diff
@@ -243,6 +247,9 @@ func (ds *SQLDatasource) handleQuery(ctx context.Context, req backend.DataQuery,
 
 // CheckHealth pings the connected SQL database
 func (ds *SQLDatasource) CheckHealth(ctx context.Context, req *backend.CheckHealthRequest) (*backend.CheckHealthResult, error) {
+	if queryDataMutator, ok := ds.driver().(CheckHealthMutator); ok {
+		ctx, req = queryDataMutator.MutateCheckHealth(ctx, req)
+	}
```

Comment:
```suggestion
	if checkHealthMutator, ok := ds.driver().(CheckHealthMutator); ok {
		ctx, req = checkHealthMutator.MutateCheckHealth(ctx, req)
	}
```

Reply:
I was thinking hmm i did that already... but it turns out i havent pushed it yet